catalog/transformer/catalog-transformer-mgmp/src/main/java/org/codice/alliance/catalog/transformer/mgmp/MgmpConstants.java

```diff
@@ -14,11 +14,299 @@
 package org.codice.alliance.catalog.transformer.mgmp;
 
 import java.util.Arrays;
+import java.util.Collections;
 import java.util.List;
 import java.util.Locale;
 
+import org.codice.ddf.spatial.ogc.csw.catalog.common.GmdConstants;
+
 public final class MgmpConstants {
```

Comment:
Some of the constants are used in both the converter and transformer, but making it package private with a comment to keep it that way sounds reasonable.